hushline/settings.py

```diff
@@ -683,4 +681,26 @@ def delete_account() -> Response | str:
         flash("User not found. Please log in again.")
         return redirect(url_for("login"))
 
+    @bp.route("/verify-rel-me/<int:field_index>", methods=["POST"])
```

Comment:
is this route used? It looked like the main settings route is responsible for verifying links in extra fields.

Reply:
I don't think this route is being used. But also it doesn't do any verification, so it should probably be removed.

Reply:
Changes in c114b3a